docs/src/examples/components/Attachment/Slots/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import ComponentExample from 'docs/src/components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/src/components/ComponentDoc/ExampleSection'
+
+const Slots = () => (
```

Comment:
one thing that am thinking about - probably, it would be nice it we would come up with some general logic of presenting these slots in a way that those will be more visible to the client